.github/workflows/release.yml

```diff
@@ -32,7 +29,7 @@ jobs:
           ref: "master"
 
       - name: Set up Go
-        uses: actions/setup-go@93397bea11091df50f3d7e59dc26a7711a8bcfbe
+        uses: actions/setup-go@v0.6
```

Comment:
change `setup-ko` not `setup-go` :D
https://github.com/ko-build/setup-ko

Reply:
oops

Reply:
from the example, it still needs setup-go no?